daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -375,13 +375,17 @@ func (l *logStream) create() error {
 		if l.logCreateGroup {
 			if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == resourceNotFoundCode {
 				if err := l.createLogGroup(); err != nil {
-					return err
+					return errors.Wrap(err, "Failed to create Cloudwatch log group")
 				}
-				return l.createLogStream()
+				err := l.createLogStream()
+				if err != nil {
+					return errors.Wrap(err, "Failed to create Cloudwatch log stream")
+				}
+				return nil
 			}
 		}
 		if err != nil {
-			return err
+			return errors.Wrap(err, "Failed to create Cloudwatch log group")
```

Comment:
Double check; should this be "log stream" instead of "log group" here?

Also (nit) errors in go should generally start with a lowercase ("failed to" instead of "Failed to"). We're not really consistent there in existing errors, so that's not a blocker 😅

Reply:
It SHOULD be log stream, thanks for taking a close look! I also updated them to make the errors lower case.